.github/workflows/publish-provider-family.yml

```
@@ -51,6 +51,7 @@ env:
   # Registry/Org names
   CROSSPLANE_REGORG: 'ghcr.io/crossplane-contrib' # xpkg.crossplane.io/crossplane-contrib
   UPBOUND_REGORG: 'xpkg.upbound.io/crossplane-contrib'
+  XPKG_CROSSPLANE_REGORG: 'xpkg.crossplane.io/crossplane-contrib'
```

```suggestion
  CROSSPLANE_PROXY_REGORG: 'xpkg.crossplane.io/crossplane-contrib'
```
(optional) - for clarity

---

Done!